src/components/CurrentSong.jsx

```diff
@@ -34,6 +61,43 @@ function CurrentSong() {
                 </div>
             </div>
 
+<<<<<<< HEAD
```

Comment:
(issue): I think this implementation of the music play buttons uses an iteration of the project where the CurrentSong didn't have the asset imports or new functionality. 

I think going back and resolving the merge to use the new functionality would be best, since that has functionality for the mobile view such as opening the CurrentSongOffCanvas.